packages/node/src/events/propose-install/operation.ts

```diff
@@ -1,18 +1,21 @@
-import { Address, Node } from "@counterfactual/types";
+import { Node } from "@counterfactual/types";
 
 import { ProposedAppInstanceInfo } from "../../models";
 import { Store } from "../../store";
 import { getChannelFromPeerAddress } from "../../utils";
 
 export async function setAppInstanceIDForProposeInstall(
-  selfAddress: Address,
+  selfAddress: string,
```

Comment:
Change variable naming I suppose since it's not an address.

Reply:
FIxing.